src/test/kotlin/no/nav/medlemskap/cucumber/steps/aareg/AaregMapperSteps.kt

```diff
@@ -0,0 +1,357 @@
+                sistStillingsendring = null,
+                skipsregister = null,
+                skipstype = String(),
+                fartsomraade = String(),
+                yrke = String(),
+                sporingsinformasjon = AaRegSporingsinformasjon(
+                    endretAv = null,
+                    endretKildeReferanse = null,
+                    endretKilde = null,
+                    endretTidspunkt = null,
+                    opprettetTidspunkt = LocalDateTime.MIN,
+                    opprettetAv = String(),
+                    opprettetKilde = null,
+                    opprettetKildereferanse = String()
+                ),
+                stillingsprosent = null,
+                sistLoennsendring = null
+            )
+
+        fun hentAaregArbeidsforhold(): AaRegArbeidsforhold =
+            AaRegArbeidsforhold(
+                ansettelsesperiode,
+                antallTimerForTimeloennet,
+                arbeidsavtaler,
+                arbeidsforholdId,
+                arbeidsgiver,
+                arbeidstaker,
+                innrapportertEtterAOrdningen,
+                navArbeidsforholdId,
+                opplysningspliktig,
+                permisjonPermitteringer,
+                registrert,
+                sistBekreftet,
+                sporingsinformasjon,
+                type,
+                utenlandsopphold
+            )
+
+        fun hentUtenlandsopphold(): AaRegUtenlandsopphold =
+            AaRegUtenlandsopphold(
+                landkode = String(),
+                rapporteringsperiode = YearMonth.now(),
+                sporingsinformasjon = hentAaregSporingsinformasjon(),
+                periode = AaRegPeriode(LocalDate.MIN, LocalDate.MAX)
+            )
+
+        fun hentOrganisasjon(): Organisasjon =
+            Organisasjon(
+                navn = navn,
+                organisasjonsnummer = null,
+                type = String(),
+                bestaarAvOrganisasjonsledd = mutableListOf<BestaarAvOrganisasjonsledd?>(),
+                inngaarIJuridiskEnheter = mutableListOf<JuridiskEnhet>(),
+                organisasjonDetaljer = Organisasjonsdetaljer(
+                    mutableListOf<Ansatte>(Ansatte(null, null, null)),
+                    null,
+                    null,
+                    mutableListOf<Enhetstyper>(Enhetstyper(null, String(), null)),
+                    null,
+                    null,
+                    null,
+                    mutableListOf<Hjemlandregistre>(),
+                    null,
+                    null,
+                    null,
+                    null,
+                    null,
+                    null,
+                    null,
+                    null,
+                    null,
+                    null,
+                    null,
+                    mutableListOf<Status>(),
+                    null,
+                    null,
+                    null,
+                    null,
```

Comment:
Veldig mange nullverdier her. Er dette data vi ikke har bruk for? Hadde det vært mulig å kutte ut å skrive så mange nuller og istedenfor gjøre noe ala:
`Organisasjonsdetaljer(
enhetstyper = mutableListOf<Enhetstyper>(Enhetstyper(enhetstyper = String()))`

Reply:
Ja, får det til på String() verdiene, men som f.eks  mutableListOf<Ansatte>(Ansatte(null, null, null)), som er int, date, date, så må jeg opprette Ansatte objektet, men får ikke til å fylle med noe bedre. Noen forslag?

Reply:
Enhetstypene er slik: mutableListOf<Enhetstyper>(Enhetstyper(null, String(), null)),

Reply:
Har  ryddet opp